app/core/db.py

```diff
@@ -0,0 +1,20 @@
+from sqlmodel import create_engine
+
+from app.core.config import settings
+from app.core.models import ExpenseBudget  # noqa: F401
+from app.core.models import ExpenseCategory  # noqa: F401
+from app.core.models import ExpenseCategoryGroup  # noqa: F401
+from app.core.models import ExpenseTransaction  # noqa: F401
```

Comment:
why do you import those modules. im sure there is another way to ensure all tables exist without this.
consider changing the SQL you are using to be less "maverick"

Reply:
What do you mean by less maverick?

I was basically following the pattern from Fast API's full stack template:
https://github.com/fastapi/full-stack-fastapi-template/blob/master/backend/app/core/db.py

Reply:
Check sqlalchemy, maybe it will make things easier (something simmilar to Orion)